packages/debugger/lib/solidity/selectors/index.js

```diff
@@ -361,17 +410,57 @@ let solidity = createSelectorTree({
    */
   views: {
     /**
-     * solidity.views.findOverlappingRange
-     * grouped by compilation
+     * solidity.views.flattenedSources
+     * all the sources just flattened into a single array
+     */
+    flattenedSources: createLeaf(["/info/sources"], compilations =>
+      flatten(
+        Object.values(
+          compilations
+        ).map(
+          ({
+            userSources: { byId: userSources },
+            internalSources: { byContext: internalSources }
+          }) =>
+            userSources.concat(
+              ...Object.values(internalSources).map(
+                ({ byId: source }) => source
+              )
+            )
+        )
+      ).filter(source => source)
+    ),
+
+    /**
+     * solidity.views.overlapFunctions
+     * grouped by compilation [then by context for internal ones]
      */
-    findOverlappingRange: createLeaf(["/info/sources"], compilations =>
+    overlapFunctions: createLeaf(["/info/sources"], compilations =>
```

Comment:
This name's a bit weird - what is an "overlap function"?

Reply:
It's a function for a given source that takes a source range and returns the AST nodes that overlap it.  Not sure what a better name would be.